apps/files/src/components/ocDialogPrompt.vue

```diff
@@ -47,6 +47,7 @@ export default {
     ocLoading: { type: Boolean, default: false },
     ocCancelId: String,
     ocConfirmId: String,
+    ocConfirmed: { type: Boolean, default: false },
```

Comment:
instead of this you can reset clicked to false within the watcher for ocActive - https://github.com/owncloud/phoenix/blob/34f06ae308c25cd6df19307101d32f0a2837bb9c/apps/files/src/components/ocDialogPrompt.vue#L71

Reply:
that's right. Why the hell did I mess around with promises and callbacks